examples/let-expression/let_expression.description

```diff
@@ -0,0 +1 @@
+// A let expression binds variables and then evaluates an expression with those variables in scope.
```

Comment:
```suggestion
// A let expression binds variables and then evaluates an expression with those variables in its scope.
```

Reply:
in scope is correct. Maybe it should be in-scope

Reply:
+1